task-management/src/main/webapp/vue-app/tasks/components/TasksApp.vue

```diff
@@ -311,6 +312,12 @@
         location.href=`${ eXo.env.portal.context }/${ eXo.env.portal.portalName }/${ pagelink }` ;
 
       },
+      retrieveTask(){
+        this.getMyOverDueTasks();
```

Comment:
Could you try to simplify this by just pushing the bnew task into the desired list of tasks without getting information from the server ?